jetty-core/jetty-http2/jetty-http2-common/src/main/java/org/eclipse/jetty/http2/internal/HTTP2Flusher.java

```diff
@@ -345,6 +345,7 @@ protected void onSuccess()
     private void finish()
     {
         accumulator.clear();
+        // TODO grab lock
```

Comment:
Is a lock really needed? `finish` is only called from `onSuccess` and `process` so shouldn't that guarantee that only a single thread can call it at any one time?